rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -374,3 +378,34 @@ template<> inline std::shared_ptr<VideoLoaderSingleShardNode> MasterGraph::add_n
 
     return node;
 }
+
+/*
+ * Explicit specialization for AudioLoaderNode
+ */
+template<> inline std::shared_ptr<AudioLoaderNode> MasterGraph::add_node(const std::vector<Tensor*>& inputs, const std::vector<Tensor*>& outputs)
+{
```

Comment:
Please change braces to google style
Applicable all such instances in this PR

Reply:
done